vignettes/arrays.Rmd

```diff
@@ -508,6 +508,50 @@ o$sub(np$arange(1, 25)$reshape(4L, 3L, 2L), np$array(y))
 The above NumPy arrays are the same, their element-wise difference is zero.
 
 
+### Reshaping arrays
+
+In R you would typically reshape an array using the `dim<-()` function. For example:
+
+```{r, eval=FALSE}
+dim(x) <- c(1000, 28, 28)
+```
+
+In R, this operation simply changes the "dim" attribute of the array, effectively 
+re-interpreting the array indices as specified using column-major semantics.
+
+However, the NumPy `reshape` method uses row-major semantics, so if you are mixing
```

Comment:
We might want to say "by default", since technically one can switch between C-style reshaping and F-style reshaping by using `order = "{C/F}"` with `np$reshape()` / `array_reshape()`. We might also want to emphasize that some Python modules (e.g. Keras) will call `np$reshape()` without specifying an ordering (or allowing the user to specify an ordering) so our hands are effectively tied and to ensure consistency we need to do this row-major-style reshaping.